scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -194,7 +197,7 @@ def export_standalone_project(ctx: exp.O3DEScriptExportContext,
                                                              expected_bundles_path / f'engine_{selected_platform}.pak'],
                                             export_layout=export_layout,
                                             archive_output_format=archive_output_format,
-                                            logger=logger)
+                                            logger=logger)      
```

Comment:
nitpick: trailing whitespace here
```suggestion
                                            logger=logger)
```